controller/localip/localip.go

```diff
@@ -186,7 +186,7 @@ func parsePodConfigFromAnnotations(annotations map[string]string, pod *podConfig
 		}
 	}
 	pod.statusPort = uint16(statusPort)
-	excludeInboundPorts := []uint16{15090, 15006, 15001, 15000, 15020} // todo changeme
```

Comment:
BTW why are we excluding 15000? I don't see this port being excluded by default, please correct me if I'm wrong